.github/workflows/sync-translations.yml

```diff
@@ -0,0 +1,141 @@
+name: Migrate translations from the old Transifex project
+
+on:
```

Comment:
it's ready for review @OmarIthawi , but I'm not sure when/how exactly would you like this action to run